specification/level-1-version-4/sources/output.tex

```diff
@@ -145,12 +144,13 @@ \subsubsection{\element{ShadedArea}}
 \end{blockChanged}
 
 
-
 % ~~~ SURFACE ~~~
 \subsubsection{\element{Surface}}
 \label{class:surface}
 \begin{blockChanged}
-A \Surface is a parallel class to \AbstractCurve that defines a three-dimensional surface instead of a two-dimensional curve (\fig{surface}).  It defines the attributes \element{xDataReference}, \element{yDataReference}, and \element{zDataReference}, all of type \SIdRef, the first two of which are optional and the last of which is required.  It also defines the optional attributes \element{style} of type \SIdRef, and \element{type}, of type \SurfaceType.
+A \Surface is a three-dimensional \Output component representing a (processed) simulation result (\fig{surface}).  A \Surface is a parallel class to \AbstractCurve that defines a three-dimensional surface instead of a two-dimensional curve.  Zero or more \Surface instances define a \PlotThree (\fig{output}).
+
+In addition to the optional \element{id} and \element{name} attributes it inherits from \SedBase (the latter of which should be used as the label in the \Plot3D legend, if present), it also defines the attributes \element{xDataReference}, \element{yDataReference}, and \element{zDataReference}, all of type \SIdRef, the first two of which are optional and the last of which is required.  It also defines the optional attributes \element{style} of type \SIdRef, and \element{type}, of type \SurfaceType.
```

Comment:
This needs to be \PlotThree instead of \Plot3D

Reply:
fixed